### ISSUE TITLE
Generated edown files missing

### ISSUE THREAD

**Issue by zuiderkwast** [uwiger/locks#9]
In README.md there are links to edown generated docs, e.g. `doc/locks.md`, but the md files are not there.

**Comment by uwiger** [comment]
True. I have fixed some minor doc-related bugs and generated and pushed the markdown.

Thanks.
